.github/workflows/pypi.yml

```
@@ -27,9 +27,9 @@ jobs:
       - uses: actions/setup-python@v2
         with:
           python-version: "3.8"
-      - name: Build the sdist
+      - name: Build the sdist and the wheel
         run: |
-          python setup.py sdist
+          python setup.py sdist bdist_wheel
```

```suggestion
          pip install wheel
          python setup.py sdist bdist_wheel
```
I think that will fix the error in CI.